static/.htaccess

```diff
@@ -1,29 +1 @@
-ExpiresActive on
 FileETag MTime
```

Comment:
@jezdez Do we still need this or will Whitenoise and/or Django take care of it from now on?